packages/local_auth/local_auth_darwin/darwin/Classes/FLALocalAuthPlugin.m

```diff
@@ -71,6 +72,14 @@ - (instancetype)initWithContextFactory:(NSObject<FLADAuthContextFactory> *)facto
   return self;
 }
 
+- (instancetype)initWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
```

Comment:
There should not be any codepaths that don't set the registrar. This class should follow standard Obj-C convention of having an init method that has every possible parameter, and any necessary convenience inits should delegate to that one.

I would not expect the `initWithContextFactory:` to be necessary any more, since tests should always be specifying all arguments, so you should be able to just add a parameter to that method.